source/dem/update_local_particle_containers.cc

```diff
@@ -83,6 +84,14 @@ update_contact_container_iterators(
                   unsigned int particle_two_id = adjacent_map_iterator->first;
                   adjacent_map_iterator->second.particle_two =
                     particle_container[particle_two_id];
+
+                  if (clear_contact_structures)
+                    {
+                      // Update the particle id of the particle in contact
```

Comment:
Is this comment still valid? I am not sure it makes sense no? (or it should apply to the line above no?)